src/signals/incident-management/containers/IncidentDetail/components/DetailHeader/index.js

```diff
@@ -88,7 +88,9 @@ const DetailHeader = ({ incident, baseUrl, onPatchIncident }) => {
     },
   };
 
-  const referrer = location.referrer || `${baseUrl}/incidents`;
+  const mapOverviewLocation = '/manage/incidents/kaart';
+
+  const referrer = location.referrer === mapOverviewLocation ? mapOverviewLocation : `${baseUrl}/incidents`;
```

Comment:
What if the referrer isn't exactly the route that you're checking for, but, for instance, contains a hash in the URL?

Reply:
It would navigate to the incidents list.

Reply:
Exactly. And that wouldn't be advisable. We still have a ticket on the board that describes having all filter, pagination and sorting params in the URL. With the current rigid check, that would fail.

Reply:
Ok, I will check only on the containing of the path then. Didn't know about that ticket.